content/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -0,0 +1,477 @@
+complete the evaluation of a parameter configuration is long.
+
+Katib uses the 
+[Scikit-Optimize](https://github.com/scikit-optimize/scikit-optimize) library
+for its Bayesian search. Scikit-Optimize is also known as `skopt`.
+
+Katib supports the following algorithm settings:
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Setting Name</th>
+        <th>Description</th>
+        <th>Example</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>base_estimator</td>
+        <td>[“GP”, “RF”, “ET”, “GBRT” or sklearn regressor, default=“GP”]: 
+          Should inherit from <code>sklearn.base.RegressorMixin</code>.
+          The <code>predict</code> method should have an optional 
+          <code>return_std</code> argument, which returns 
+          <code>std(Y | x)</code> along with <code>E[Y | x]</code>. If 
+          <code>base_estimator</code> is one of 
+          [“GP”, “RF”, “ET”, “GBRT”], the system uses a default surrogate model
+          of the corresponding type. See more information in the
+          <a href="https://scikit-optimize.github.io/#skopt.Optimizer">skopt
+          documentation</a>.</td>
+        <td>GP</td>
+      </tr>
+      <tr>
+        <td>n_initial_points</td>
+        <td>[int, default=10]: Number of evaluations of <code>func</code> with 
+          initialization points before approximating it with 
+          <code>base_estimator</code>. Points provided as <code>x0</code> count
+          as initialization points. 
+          If <code>len(x0) &lt; n_initial_points</code>, the
+          system samples additional points at random. See more information in the
+          <a href="https://scikit-optimize.github.io/#skopt.Optimizer">skopt
+          documentation</a>.</td>
+        <td>10</td>
+      </tr>
+      <tr>
+        <td>acq_func</td>
+        <td>[string, default=<code>&quot;gp_hedge&quot;</code>]: The function to
+          minimize over the posterior distribution. See more information in the
+          <a href="https://scikit-optimize.github.io/#skopt.Optimizer">skopt
+          documentation</a>.</td>
+        <td>gp_hedge</td>
+      </tr>
+      <tr>
+        <td>acq_optimizer</td>
+        <td>[string, “sampling” or “lbfgs”, default=“auto”]: The method to 
+          minimize the acquistion function. The system updates the fit model
+          with the optimal value obtained by optimizing <code>acq_func</code>
+          with <code>acq_optimizer</code>. See more information in the
+          <a href="https://scikit-optimize.github.io/#skopt.Optimizer">skopt
+          documentation</a>.</td>
+        <td>auto</td>
+      </tr>
+      <tr>
+        <td>random_state</td>
+        <td>[int]: Set <code>random_state</code> to something other than None
+          for reproducible results.</td>
+        <td>10</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<a id="hyperband"></a>
+#### HYPERBAND
```

Comment:
Why is this one in all caps?

Reply:
Because that's how [the paper](https://arxiv.org/pdf/1603.06560.pdf) refers to it throughout its text.

Reply:
The paper uses upper and lower-case letters for Hyperband in the title and headings. The all-caps notation is probably just formatting.

Reply:
I checked the paper carefully before I changed the name to caps. As far as I could see, the paper uses all caps almost always, so I decided it was a conscious branding decision. Let's go with all caps.